tests/pytorch/test_pytorch_autolog.py

```diff
@@ -57,6 +61,20 @@ def pytorch_model_with_steps_logged(request):
     return trainer, run, log_every_n_epoch, log_every_n_step
 
 
+@pytest.fixture(params=[(1, 1), (1, 10), (2, 1)])
```

Comment:
In order to detect the issue that I mentioned about the rounding behavior, can these fixture parameters be adjusted so that there is an uneven combination that would result in rounding estimations that would create repeated log entries?